CHANGELOG.md

```diff
@@ -127,8 +127,8 @@
 - Force usage of `receiverhelper.NewFactory` to implement `component.ReceiverFactory` (#4338)
 - Force usage of `extensionhelper.NewFactory` to implement `component.ExtensionFactory` (#4338)
 - Move `service/parserprovider` package to `config/configmapprovider` (#4206)
-   - Rename `MapProvider` interface to `Provider`
-   - Remove `MapProvider` from helper names
+  - Rename `MapProvider` interface to `Provider`
```

Comment:
Are changes to old version intentional?

Reply:
No, this was my helpful "format on-save" feature. Reverted.